ccdproc/tests/test_image_collection.py

```diff
@@ -1100,3 +1100,8 @@ def test_filtered_collection_with_no_files(self, triage_setup):
         ifc = ImageFileCollection(triage_setup.test_dir)
 
         ifc_no_files = ifc.filter(object='really fake object')
+
+    def test_filter_with_parenthetical_keyword_values(self, triage_setup):
+        # Would need test data containing FITS keywords whose values have ()
+        # e.g.:  FILTREAR= 'Clear (C)'          / Rear Filter
+        pass
```

Comment:
I think the short example that @mkelley provided in #769 could be used as the body of the test here, or at least as the start of a test. It generates a FITS file with a special character in it then tries to filter by a value with the special character. 

I can add a credit to @mkelley for the commit with the test body if you end up using a lot of his sample code.

Reply:
Yes, I will add this test this week.  Change to use re.escape() already committed.

Reply:
Excellent, thanks!